tests/e2e/simple-app.test.ts

```diff
@@ -12,8 +12,8 @@ test('Renders the Home page correctly', async ({ page, simple }) => {
 
   await expect(page).toHaveTitle('Simple Next App')
 
-  expect(headers['cache-status']).toMatch(/^"Next.js"; hit$/m)
-  expect(headers['cache-status']).toMatch(/^"Netlify Edge"; fwd=miss$/m)
+  expect(headers['cache-status'].replaceAll(', ', '\n')).toMatch(/^"Next.js"; hit$/m)
+  expect(headers['cache-status'].replaceAll(', ', '\n')).toMatch(/^"Netlify Edge"; fwd=miss$/m)
```

Comment:
Not related to main portion of PR - seems like we might be getting comma separated cache-status values, so I'm massing it to have each statement as separate line to match regex assumption